panel/widgets/chat.py

```diff
@@ -35,18 +35,42 @@
 from .indicators import LoadingSpinner
 from .input import FileInput, TextInput
 
+ASSISTANT_LOGO = "🤖"
 GPT_3_LOGO = "https://upload.wikimedia.org/wikipedia/commons/thumb/0/04/ChatGPT_logo.svg/1024px-ChatGPT_logo.svg.png?20230318122128"
 GPT_4_LOGO = "https://upload.wikimedia.org/wikipedia/commons/a/a4/GPT-4.png"
+SYSTEM_LOGO = "⚙️"
+USER_LOGO = "🧑"
```

Comment:
We can also leave it with the smiling emoji.

This emoji of a genderless person also smiles. But then it signals a human as well. I think that is a positive. So at least I would like us to consider if its a beter emoji.

Let me know. Thanks.